### DISCUSSION TITLE
Fragment in Outbounds Configuration

### DISCUSSION THREAD

**Discussion by ILYAGVC** [XTLS/Xray-core#3521] [Q&A]
Is it possible to enable the fragment in Xray Outbounds Configuration?

**Comment by Morrigan-Ship** [comment]
fake_packets: 40-80
fake_packets_mode: m1, m2, m3, m4, m5, m6
fake_packets_delay: 4-8
fake_packets_size: 40-100

Hiddify is supporting fragment
hiddify.com is a fork of sing-box that supporting fragment
